androidApp/src/main/res/values/strings.xml

```diff
@@ -1,3 +1,11 @@
 <resources>
-    <string name="app_name">Press</string>
+  <string name="app_name">Press</string>
+
+  <item name="homeViewId" type="id" />
+  <item name="editorViewId" type="id" />
+
+  <!-- Unique integers to be used in EditorView to store some transient states -->
+  <item name="scrollStatePosition" type="id" />
+  <item name="scrollStateLineDiff" type="id" />
+  <item name="scrollStateLineOffset" type="id" />
```

Comment:
This file should only be used for strings. Can you move these to `ids.xml`?

Reply:
Agreed.